ember-model.js

```diff
@@ -4,205 +4,224 @@
 // License:   Licensed under MIT license (see license.js)
 // ==========================================================================
 
-// 0.0.14-78-g1a35b19
-// 1a35b19 (2019-04-10 09:36:18 +0100)
+// 3.8.0-intercom.1-1-g362dcfb
```

Comment:
3.10-intercom...? Not sure how this is generated but you may need to update the package.json version number too.

Reply:
thanks, just pushed an update

Reply:
Do you need to rebuild the output or something to get this version number to update?

Reply:
I'm not sure where this is pulled from. I updated it manually as it's just a comment